bro_connector/frd/models.py

```diff
@@ -290,4 +290,31 @@ class FormationresistanceRecord(FRDRecord):
     class Meta:
         managed = True
         db_table = 'frd"."formationresistance_record'
-        verbose_name_plural = "Formationresistance Records"
\ No newline at end of file
+        verbose_name_plural = "Formationresistance Records"
+
+class FrdSyncLog(models.Model):
+    synced = models.BooleanField(default=False)
+    date_modified = models.CharField(max_length=254, null=True, blank=True)
+    event_type = models.CharField(
+        choices=EVENT_TYPE_CHOICES,
+        blank=False,
+        null=True,
+        max_length = 25,
+    )
+    frd = models.ForeignKey(FormationResistanceDossier, on_delete = models.CASCADE, null = True, blank = True)
+    frd_bro_id = models.CharField(max_length=254, null=True, blank=True)
+    process_status = models.CharField(max_length=254, null=True, blank=True)
+    comment = models.CharField(max_length=10000, null=True, blank=True)
+    xml_filepath = models.CharField(max_length=254, null=True, blank=True)
```

Comment:
Wat is dit path? Absoluut? Relatief? Relatief t.o.v. wat?
Is 254 genoeg? De folderstructuur op de projectenschijf van N&S gaat soms OPS-tekenkrommend diep :-)

Reply:
Sowieso lastig punt qua naamgeving (vind ik vaak zelf). "file", "filename", "path", "filepath"? Welke gebruik je?
Filename is een losse filename, bij een path zitten er directories bij. Maar ja, is dat dan path of filepath :-)
Vaak gebruik ik "file" als een geöpend file object en filename/filepath als de str/Path.
Enfin..... zolang het maar intern een beetje consistent is.